VideoApi/VideoApi/Validations/AddQuickLinkParticipantRequestValidation.cs

```diff
@@ -1,16 +1,21 @@
 using FluentValidation;
 using VideoApi.Contract.Requests;
+using System.Text.RegularExpressions;
 
 namespace VideoApi.Validations
 {
     public class AddQuickLinkParticipantRequestValidation : AbstractValidator<AddQuickLinkParticipantRequest>
     {
         public static readonly string NoNameErrorMessage = "Name is required";
         public static readonly string NoUserRoleErrorMessage = "UserRole is required";
+        public static readonly string SpecialCharNameErrorMessage = @"Name must not contain the following characters ! ” # $ % & ( ) * + , . / : ; < = > ? @ [ \ ] ^ _ ` { | } ~";
+
+        public static readonly Regex RegexName = new Regex(@"^[^!”#$%&()*+,./:;<=>?@[\\\]^_`{|}~]+$");
```

Comment:
are we using this property anywhere, from what I can see it only appears to be used here, if that is the case can we change this to a string as we are converting this back to string?

Reply:
done now